setup.py

```diff
@@ -569,15 +573,31 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
     include = data.get("include", [])
     include.append(numpy.get_include())
 
+    extra_comp_args = extra_compile_args.copy()
+    comp_macros = data.get("macros", macros)
+    undef_macros = []
+
+    if is_platform_zos():
```

Comment:
I don't see why we need this function, why not simply `if sys.platform == "zos":`?

Reply:
I had added it to be consistent with how the other platforms handled it. I've minimized the change as suggested.